ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/clientlibs/site/js/checkboxgroupview.js

```diff
@@ -97,6 +98,15 @@
                     this.setInactive();
                 });
             })
+
+            let optionLinks = this.element.querySelectorAll(`.${CheckBoxGroup.selectors.optionLink}`);
+            optionLinks.forEach(optionLink => {
+                optionLink.addEventListener('keydown', (e)=>{
+                    if(e.key === 'Enter' || e.key ===' ') {
```

Comment:
Create a variable for the list ['Enter', ' ', 'Spacebar', 'Space'] and use it wherever needed. Please update it in the link below.
https://github.com/adobe/aem-core-forms-components/pull/1591#discussion_r2199698322